File: Day1/26-Remove-Duplicates-from-Sorted-Array/Method1.cpp
class Solution
{
public:
    int removeDuplicates(vector<int> &nums)
    {
        int j = 0, count = 1;

        for (int i = 1; i < nums.size(); i++)
        {
            if (nums[i] == nums[j])
                continue;
            else
            {
                j++;
                count++;
                nums[j] = nums[i];
            }
        }

        nums.resize(count);
        return count;
    }
};

File: Day2/1-Two-Sum/Method1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer>hs=new HashMap<>();

        for(int i=0;i<nums.length;i++){
            if(hs.containsKey(target-nums[i])){
                int index=hs.get(target-nums[i]);
                if(index!=i) return new int[]{index,i};
            }
            hs.put(nums[i],i);
        }

        return new int[]{-1,-1};
    }
}